README.md

```diff
@@ -1,71 +1,59 @@
-# yandex_mapkit_lite
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/54618146/b1ca707b-b162-43d5-a9b1-d1895ab99e14">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/54618146/f88243c6-26f0-4491-93bd-baf4ad5a5e50">
+  <img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/54618146/f88243c6-26f0-4491-93bd-baf4ad5a5e50">
+</picture>
 
 [![Build Status](https://shields.io/github/actions/workflow/status/surfstudio/yandex-mapkit-lite-flutter/main.yml?logo=github&logoColor=white)](https://github.com/surfstudio/yandex-mapkit-lite-flutter)
 [![Coverage Status](https://img.shields.io/codecov/c/github/surfstudio/yandex-mapkit-lite-flutter?logo=codecov&logoColor=white)](https://app.codecov.io/gh/surfstudio/yandex-mapkit-lite-flutter)
 [![Pub Version](https://img.shields.io/pub/v/yandex_mapkit_lite?logo=dart&logoColor=white)](https://pub.dev/packages/yandex_mapkit_lite)
 [![Pub Likes](https://badgen.net/pub/likes/yandex_mapkit_lite)](https://pub.dev/packages/yandex_mapkit_lite)
 [![Pub popularity](https://badgen.net/pub/popularity/yandex_mapkit_lite)](https://pub.dev/packages/yandex_mapkit_lite/score)
 ![Flutter Platform](https://badgen.net/pub/flutter-platform/yandex_mapkit_lite)
+[![License: MIT](https://img.shields.io/badge/license-MIT-purple.svg)](https://opensource.org/licenses/MIT)
 
-## Description
+---------
 
-**This library is a fork of the package yandex_mapkit 3.4.0.**
+Fork of [yandex_mapkit](https://pub.dev/packages/yandex_mapkit) library, but less heavy and more optimized.
 
-**yandex_mapkit** is based on MapKit full version (https://yandex.ru/dev/mapkit/doc/en/).
+Made by [Surf :surfer:](https://surf.dev/flutter/) Flutter team :cow2:
 
-**yandex_mapkit_lite** is based on MapKit lite version.
+## Description
 
-Using the lite version of MapKit allows you to reduce the application size by an average of 30%.
+- :earth_africa: Map overview - enables to view the map of the world, with which user can interact with any convinient way, usually in order to demonstrate the location of some place
+- :house: Custom map objects - enables for developers to add custom map objects in order to indicate some place on the map
+- :video_game: Convinient map controls - there is an API for straight-to-point map controls through the code - from zooming and moving to limiting user scroll and controlling the speed
+- :leaves: App bundle size reduction - the average bundle size reduction for 25% comparing to projects with [original package](https://pub.dev/packages/yandex_mapkit)
```

Comment:
> bundle size reduction for 25% comparing to projects with original package

Does the size of the map increase proportionally to the size of the application?

Reply:
I believe that no. I will think how to paraphrase it.

Reply:
✅